src/main/java/storage/FileManager.java

```diff
@@ -639,7 +639,7 @@ public void addFilePlanToList(PlanList planList, String[] planFileDataLine)
     public void addFileScheduleToList(DayList dayList, String[] scheduleFileDataLine)
             throws ArrayIndexOutOfBoundsException, InvalidScheduleException, InvalidPlanException {
         String dayNumber = scheduleFileDataLine[0];
-        String planName = scheduleFileDataLine[1];
+        String planName = scheduleFileDataLine[1].toLowerCase().toLowerCase();
```

Comment:
Is there supposed to have 2 `.toLowerCase()`?

Reply:
Thanks for spotting it. It has been fixed in Commit 95f1266.